gcomm/src/defaults.cpp

```diff
@@ -63,6 +63,7 @@ namespace gcomm
     std::string const Defaults::PcVersion               = "0";
     std::string const Defaults::PcWaitPrim              = "true";
     std::string const Defaults::PcWaitPrimTimeout       = "PT30S";
+    std::string const Defaults::PcWaitRestoredPrimTimeout = "PT0S";
```

Comment:
Can we keep the default value the same as that of `pc.wait_prim_timeout`? Why to leave it as-is in default configuration when we know that this will wait forever?

Reply:
As we discussed with @altmannmarcelo 30s may be (and probably is in most cases) not enough for cluster to recover after the outage. It is hard to say a-priori what is the proper timeout and the default behavior "wait until it is done" is proper behavior for most cases (see the problem description in Jira)
Having default to wait infinitely is to keep the current behavior, which works in the vast majority of cases. Also some existing systems may rely on this behavior.

Reply:
Got it. Thanks for the clarification.

Can we add a message/warning before we wait to tell users that it will wait forever until it reaches PC and to consider setting `pc.wait_restored_prim_timeout` in case if it has to error out if failed to receive PC.